requirements/requirements.in

```diff
@@ -12,6 +12,7 @@ oauthlib
 psycopg2
 pycryptodomex
 PyJWT[crypto]
+python-jose[crypto]
```

Comment:
After we move the reminding JWT bits to the service we could see if its possible to settle only on one JWT library.

Reply:
What's wrong with the existing one? I suspect adding a new JWT library would get the Sean upside-down smile of approval.